src/components/menu/menu.types.ts

```diff
@@ -57,4 +57,28 @@ export interface MenuItem<T = any> {
      * Value of the menu item.
      */
     value?: T;
+
+    /**
+     * What sub items the item contains
+     * Don't set if using lazy loading.
+     */
+    subItems?: Array<MenuItem<T>>;
+
+    /**
+     * What parent the item has.
+     * It's used to render the breadcrumbs history
+     * Mostly handled by the menu itself
+     */
+    parentItem?: MenuItem<T>;
```

Comment:
Not sure I understand why this is needed. If a consumer does not need it we should put it in a private type instead that is not exported

Reply:
Yeah we weren't 100% happy with this solution, but the reason it's public is because that is the property that is used to build the "history" using the breadcrumb component.

So again in our hierarchy picker it's possible to open the menu that should "start" in a sub menu (not in the root but maybe in level 2 or 3) and this is then how you can visualize that from the outside.

We're very open for suggestion/discussions about this solution.

Reply:
Maybe we should have a discussion regarding this as well.

Reply:
Since there is still no example of how this is supposed to be used, can we perhaps mark it as `@private` so a consumer does not use it by mistake?

Reply:
We added an example now. I must've totally forgotten about it after our meeting.